AudioCompression/src/audioCompression/algorithm/FilterBankStep.java

```diff
@@ -82,6 +82,8 @@ public RawAudio reverse(Subbands input) {
 		}
 		WavAudioOutput out = 
 				new WavAudioOutput(windows, input.getWindowOverlap(), input.getSampleRate());
+		
+		out.writeFile(name);
```

Comment:
For instance, if we wanted to stick some sort of additional pre-processing to come in before the filterbank, the writing here would complicate that...

On the other hand, the far end of the pipeline has the very definite `SerializationStep`, which makes sense to write a file in the forward direction and read one in reverse...